src/ensembl/src/shared/components/selected-species/SelectedSpeciesDisplayName.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { getDisplayName } from './selectedSpeciesHelpers';
+
+import { CommittedItem } from 'src/content/app/species-selector/types/species-search';
+
+import styles from './selected-species-common.scss';
+
+type Props = {
+  species: CommittedItem;
+  nameClassName?: string;
+  assemblyClassName?: string;
```

Comment:
Do we not usually use a single classNames object with multiple keys?

Reply:
Switched from two independent props to `classNames` object.